.github/workflows/test.yml

```diff
@@ -24,7 +24,8 @@ jobs:
 
       - name: Install Dependencies
         run: ./setup-ci.sh
-
+      - name: Compile core contracts
+        run: make compile-core-contracts
```

Comment:
Is it because of this `TestPerformExit`? Could we somehow refactor that test to make it any smaller (and do we need it in the first place, since we have E2E test)?

Reply:
I commited artifacts inside codebase, so i got rid of this.